lib/pgp-secret.ts

```diff
@@ -61,6 +61,7 @@ interface PGPSecretProps {
  */
 export class PGPSecret extends cdk.Construct {
   public readonly secretArn: string;
+  public readonly parameterName: string;
```

Comment:
Shouldn't we call this `publicKeyParameterName`?

Reply:
No - in fact I'll even be adding a shared interface for `PGPSecret` and `PrivateKey` (from `code-signing`) to make usage of those types of secrets more consistent (that's particularly handy when you need to share those across stacks in  a streamlined manner).